attributes/jenkins.rb

```diff
@@ -19,3 +19,8 @@
 
 # When set to true a postfix SMTP server will be configured for use by Jenkins via sendmail.
 default['ros_buildfarm']['smtp'] = false
+
+# Last version supporting Java 8
+#default['jenkins']['master']['version'] = '2.346.1'
```

Comment:
This is a bit of a carry on from where this change originated (#117) but I'm interested in leaving it in as a signifier for other build farm maintainers. I expect that a PR and release between now and Java 11 support will raise our "high water level" to 2.346.1.